frontend/packages/console-shared/src/components/formik-fields/EnvironmentField.tsx

```diff
@@ -26,7 +26,7 @@ const EnvironmentField: React.FC<EnvironmentFieldProps> = ({
   const { t } = useTranslation();
   const fieldId = getFieldId(props.name, 'env-input');
   const environmentVariables = React.useMemo(() => {
-    return !_.isEmpty(envs) ? envs.map((env) => _.values(env)) : [['', '']];
+    return _.isEmpty(envs) ? [['', '']] : envs.map((env) => _.values(env));
```

Comment:
Nothing bad, just it increases the code diff. 
Should be avoided if not very necessary.
But we can move forward with ur change this time :+1:

Reply:
in my defense it took me a few minutes to parse out the original code and I wasn't going to open a PR to refactor one line :p